src/main/java/com/gmail/inverseconduit/bot/DefaultBot.java

```diff
@@ -58,7 +61,10 @@ private void processMessage(final ChatMessage chatMessage) {
         final String trigger = AppContext.INSTANCE.get(BotConfig.class).getTrigger();
         if ( !chatMessage.getMessage().startsWith(trigger)) { return; }
 
-        commands.stream().filter(c -> chatMessage.getMessage().replace(trigger, "").startsWith(c.getName())).findFirst().map(c -> c.execute(chatMessage))
+        final String commandName = chatMessage.getMessage().replace(trigger, "").split(" ")[0];
```

Comment:
That in turn would make it quite moot to use an anchor and the code then would be:

    chatMessage.getMessage().replaceFirst(Pattern.quote(trigger), "").split(" ")[0];